src/sync/commits.ts

```diff
@@ -22,26 +21,22 @@ const fetchCommits = async (gitHubClient: GitHubInstallationClient, repository:
 export const getCommitTask = async (
 	logger: Logger,
 	gitHubClient: GitHubInstallationClient,
-	jiraHost: string,
+	_jiraHost: string,
```

Comment:
Looks like `getCommitTask` is only being used in sync/installation. If we don't need `jiraHost` anymore can we just remove this param?